crates/path/src/events.rs

```diff
@@ -39,13 +39,13 @@ pub type IdEvent = Event<EndpointId, ControlPointId>;
 
 impl<Ep, Cp> Event<Ep, Cp> {
     pub fn is_edge(&self) -> bool {
-        match self {
+        matches!(
+            self,
             &Event::Line { .. }
-            | &Event::Quadratic { .. }
-            | &Event::Cubic { .. }
-            | &Event::End { close: true, .. } => true,
-            _ => false,
-        }
+                | &Event::Quadratic { .. }
```

Comment:
I'm not very fond of this extra indentation.

Reply:
I can't really get rid of the indentation unless we get rid of `cargo fmt` altogether, which I'd advise against. I could turn this back into a `match` statement and add an `#[allow(clippy::match_like_matches_macro)]` attribute, though.